api/passthrough/v1/passthrough.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+package clutch.passthrough.v1;
+
+option go_package = "github.com/lyft/clutch/backend/api/passthrough/v1;passthrough";
+
+import "api/v1/annotations.proto";
+import "google/protobuf/struct.proto";
+import "google/api/annotations.proto";
+import "validate/validate.proto";
+
+// The PassthroughAPI enabling calling external API's without minimal setup.
+// The services must be configured before calling external API's.
+service PassthroughAPI {
+  rpc GetPassthrough(GetPassthroughRequest) returns (GetPassthroughResponse) {
+    option (google.api.http) = {
+      post : "/v1/passthrough/get"
```

Comment:
`get` is confusing given that it's also and HTTP method. i would call it `rpc` or `request` or something. or is this the HTTP method? since i don't see that in the request.

Reply:
Yeah I was debating between having the HTTP method passed in or specific RPC for each, passing in the method would simplify / reduce the numbers of `rpc` definitions. I think ill move the method info to the request..